api/views/CloudRecordSummaryView.py

```diff
@@ -113,7 +124,14 @@ def get(self, request, format=None):
 
         cursor = database.cursor(MySQLdb.cursors.DictCursor)
 
-        if group_name is not None:
+        if global_user_name is not None:
```

Comment:
Is this the case that is run when user and group name are both provided? Should that be checked first and made an error?